rest_framework-stubs/views.pyi

```diff
@@ -81,10 +81,10 @@ class APIView(View):
     def determine_version(
         self, request: Request, *args: Any, **kwargs: Any
     ) -> Tuple[Optional[str], Optional[BaseVersioning]]: ...
-    def initialize_request(self, request: Request, *args: Any, **kwargs: Any) -> Request: ...
+    def initialize_request(self, request: HttpRequest, *args: Any, **kwargs: Any) -> Request: ...
     def initial(self, request: Request, *args: Any, **kwargs: Any) -> None: ...
     def finalize_response(self, request: Request, response: Response, *args: Any, **kwargs: Any) -> Response: ...
     def handle_exception(self, exc: Exception) -> Response: ...
     def raise_uncaught_exception(self, exc: Exception) -> NoReturn: ...
-    def dispatch(self, request: Request, *args: Any, **kwargs: Any): ...  # type: ignore[override]
+    def dispatch(self, request: HttpRequest, *args: Any, **kwargs: Any): ...
```

Comment:
What do we return here?

Reply:
Adjusted to `HttpResponseBase` from what I could see in the implementation of it (which also made me realize that this is wrong in `django-stubs` - it types the return value for `dispatch` as `HttpResponse`, which prevents `StreamingHttpResponse` from being a valid value...